specify/specify7
# Truncate value fields for Audit Log entries

Fixes #829 

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions
- Create or find any sufficiently large ViewSet (test sets can be download from: https://github.com/specify/specify7/issues/829#issuecomment-1915309392 and https://github.com/specify/specify7/issues/829#issuecomment-1966921788)
- Paste or load the contents of the large ViewSet into an already existing ViewSet
- The ViewSet should save correctly without throwing an error
